aptos-move/move-examples/fungible_asset/sources/fungible_source.move

```diff
@@ -0,0 +1,268 @@
+module fungible_asset::fungible_source {
+
+    use aptos_framework::object::{object_address, Object, ExtendRef, address_from_extend_ref};
+    use std::option::Option;
+    use std::option;
+    use std::error;
+    use fungible_asset::fungible_asset::{mint, set_frozen_flag, withdraw_internal, burn, deposit};
+    use aptos_framework::object;
+    use std::signer;
+    #[test_only]
```

Comment:
nit: Move these closer to the tests

Reply:
This is what IDE auto-inserted... unless we change the ide plugin behavior.